unified-scheduler-pool/src/lib.rs

```diff
@@ -955,11 +955,20 @@ impl<TH: TaskHandler> InstalledScheduler for PooledScheduler<TH> {
         &self.context
     }
 
-    fn schedule_execution(&self, &(transaction, index): &(&SanitizedTransaction, usize)) {
+    fn schedule_execution(
+        &self,
+        &(transaction, index): &(&SanitizedTransaction, usize),
+    ) -> ScheduleResult {
         let task = SchedulingStateMachine::create_task(transaction.clone(), index, &mut |pubkey| {
             self.inner.usage_queue_loader.load(pubkey)
         });
         self.inner.thread_manager.send_task(task);
+        // Return hard-coded Ok for now; Upcoming pr will implement this properly.
+        Ok(())
+    }
+
+    fn recover_error_after_abort(&mut self) -> TransactionError {
+        todo!("in later pr...");
```

Comment:
I am not a fan of letting `todo!` into the master branch. I'd much rather seen the error recovery code in this PR. OR in some initial PR with dead_code, and then this PR simply uses those changes.

It's too easy to forget todo! as a reviewing, since github's view is often limited

Reply:
> I am not a fan of letting `todo!` into the master branch. I'd much rather seen the error recovery code in this PR. 

hmm, how about closing this pr and switching to review #1211? As the pr is the superset of this pr, there's no `todo!()` there. Or, ...

> OR in some initial PR with dead_code, and then this PR simply uses those changes.

... if that pr isn't acceptable to review in one go for you, I can chunk the pr according to this.

> It's too easy to forget todo! as a reviewing, since github's view is often limited

I think we can leave some check-boxes in the pr description if it works not to forget. 

imo, explicit `todo!()` isn't so different from implicit (undocumented-but-definitely-existing) leak sources in master. And the remaining todos is existing only in my mind at the moment... ;) speaking of it, i can dump them somewhere and maintain it if it's helpful.

Reply:
alright, that's fine. let's just move to #1211